scripts/linters/custom_eslint_checks/rules/e2e-action-exclude.json

```diff
@@ -0,0 +1,62 @@
+{
+    "exclude": [
+        "ClassroomPage.js",
```

Comment:
Why are these excluded?

Reply:
Because they haven't been fixed yet. They will be removed as we fix tasks in #10798